lib/zclient.c

```diff
@@ -41,8 +41,9 @@ static void zclient_event(enum zclient_event, struct zclient *);
 static void zebra_interface_if_set_value(struct stream *s,
 					 struct interface *ifp);
 
-struct zclient_options zclient_options_default = {.receive_notify = false,
-						  .synchronous = false};
+struct zclient_options zclient_options_default = {
```

Comment:
If I am testing via bisect, won't this change cause and assert and crash in zebra every time?  This makes me unhappy

Reply:
???

Why would this crash anything?

Reply:
In this commit the byte of data sent in the send_hello function is removed.  The equivalent code in zebra to read this will now crash as that it will try to read 1 byte beyond the end of the stream, as that we have code that intentionally crashes when this happens.  As I stated if this commit is choosen as the bisect I will be stopped in my tracks if I attempt to run zebra with any other daemon at all to find a different problem.  This is especially egregious from my perspective because I cannot work around this issue at all as that the initial connection from any daemon will cause zebra to crash

Reply:
The byte is not removed in this commit. Please flush your brain cache and look at the commit again :laughing:
https://github.com/FRRouting/frr/pull/14867/commits/25a1dccc56a7e544a855bfeda5751c91b53a03fb#diff-f949c6f0ae3101476f0699de68a4a73935fdd945294d5f235e97fb5e8bdbc782L395-R395